### PR TITLE
Add support for tr

### PR DESCRIPTION
**Pull request by ehedrick**
This is for #288

### CODE REVIEW

File: lib/page-object/accessors.rb [hunk 1]
```diff
@@ -637,6 +637,36 @@ def cell(name, identifier={:index => 0}, &block)
...
+    #   * :class => Watir and Selenium
+    #   * :css => Watir and Selenium
+    #   * :id => Watir and Selenium
+    #   * :index => Watir and Selenium
+    #   * :text => Watir only
+    #   * :xpath => Watir and Selenium
+    #   * :css => Watir and Selenium
+    # @param optional block to be invoked when element method is called
+    #
+    def tr(name, identifier={:index => 0}, &block)
```
Comment (daneandersen) [inline lib/page-object/accessors.rb]: The corresponding method for td elements is cell. To keep with the past pattern, this should probably be `row`

File: lib/page-object/platforms/selenium_webdriver/page_object.rb [hunk 2]
```diff
@@ -576,6 +576,31 @@ def cells_for(identifier)
         end
 
         #
+        # platform method to retrieve the text from a table row
+        # See PageObject::Accessors#tr
+        #
+        def tr_text_for(identifier)
```
Comment (daneandersen) [inline lib/page-object/platforms/selenium_webdriver/page_object.rb]: As above the corresponding td methods are `cell_text_for`, `cell_for`, and `cells_for`, so these should probably be `row_text_for`, `row_for`, and `rows_for`

**Comment by daneandersen** [comment]
This looks good. Do you want to make the changes I listed? After that I can merge it.

**Comment by ehedrick** [comment]
I was going back and forth on that one myself. I updated everything to sue row instead or tr.

**Pull request merged**